libcloudforensics/providers/gcp/internal/monitoring.py

```diff
@@ -95,3 +95,82 @@ def ActiveServices(self, timeframe: int = 30) -> Dict[str, int]:
             if val:
               ret[service] = int(val)
     return ret
+
+  def _BuildCpuUsageFilter(self, instances: Optional[List[str]]) -> str:
+    """Builds a metrics query filter based on a list of instances.
+
+    Args:
+      instances list[str]: a list of instance names.
+
+    Returns:
+      str: the filter to use in a metrics query.
+    """
+    instances_filter = (
+          ['metric.type = "compute.googleapis.com/instance/cpu/utilization"'])
+    if instances:
+      instances_filter.append(
+          ' AND (metric.label.instance_name = "{0:s}"'.format(instances[0]))
+      if len(instances) > 1:
+        for instance_name in instances[1:]:
+          instances_filter.append(
+              ' OR metric.label.instance_name = "{0:s}"'.format(instance_name))
+      instances_filter.append(')')
+
+    return ''.join(instances_filter)
+
+
+  def GetCpuUsage(self, instances: Optional[List[str]] = None, days: int = 7,
+      aggregation_minutes: int = 60) -> Dict[str, List[Tuple[str, float]]]:
```

Comment:
```suggestion
  def GetCpuUsage(self,
    instances: Optional[List[str]] = None,
    days: int = 7,
    aggregation_minutes: int = 60
    ) -> Dict[str, List[Tuple[str, float]]]:
```
I believe this is the preferred format for multi-line method declarations.